spec.html

```diff
@@ -10927,7 +10863,7 @@ <h1>Module Environment Records</h1>
           <h1>
             GetBindingValue (
               _N_: a String,
-              _S_: a Boolean,
+              _S_: *true*,
```

Comment:
Ehhh, I don't like this change. This is an implementation of an abstract method, and the abstract method takes a boolean here, not `*true*`. I'd prefer to keep the assert.

Reply:
Yes but this concrete variant is only ever passed `true`. If we wanted all variants of this method to have the same signature, we should have a centralised description of it.

Reply:
We do, in [table 17](https://tc39.es/ecma262/multipage/executable-code-and-execution-contexts.html#table-abstract-methods-of-environment-records).

Reply:
I mean a centralised signature, with all the metadata.

Reply:
That seems arguably useful, but I don't see what bearing it has on this question. Overloads generally have the same signature as the method they're overloading. This should as well. Please put it back.